web/src/components/button.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'gatsby';
+
+export default function Button({
+    isPrimary,
+    isPrimaryDark,
+    isSecondary,
+    isSecondaryDark,
+}) {
+    const primaryStyle =
+        'bg-black text-white hover:bg-brand-1 active:bg-gray-300';
+    const primaryDarkStyle =
+        'bg-white text-black hover:bg-gray-300 active:bg-brand-1';
+    const secondaryStyle =
+        'bg-white text-black hover:bg-brand-1 hover:text-brand-1 active:text-gray-300';
+    const secondaryDarkStyle =
+        'bg-brand-1 text-white hover:text-gray-300 active:text-brand-1';
```

Comment:
Don't need active states. Please add `focus:` classes that match `hover:`

Reply:
@asasmith oki doki